ext/opentelemetry-ext-jaeger/tests/test_jaeger_exporter.py

```diff
@@ -209,6 +220,17 @@ def test_translate_to_jaeger(self):
                         vType=jaeger.TagType.DOUBLE,
                         vDouble=111.22,
                     ),
+                    jaeger.Tag(
+                        key="status.code", vType=jaeger.TagType.LONG, vLong=2,
```

Comment:
Is it possible to change the `2` by `StatusCanonicalCode.UNKNOWN`?

Reply:
Sure! It's better to see the enum than a random name.